zerver/models.py

```diff
@@ -1369,7 +1366,7 @@ def linkifiers_for_realm_remote_cache(realm_id: int) -> List[LinkifierDict]:
         linkifiers.append(
             LinkifierDict(
                 pattern=linkifier.pattern,
-                url_format=linkifier.url_format_string,
+                url_template=linkifier.url_template,
```

Comment:
This is an API change that we'll need to manage. First, we need to discuss in "api design" how to avoid breaking the mobile apps with this transition -- one possibility might involve try to construct a backwards-compatibility `url_format` value to send to clients where possible, but it's very possible we'll need to do something fancier involving sending the empty list or something -- we don't want existing clients to crash due to the `url_format` field being missing.

Second, we need to document whatever exact API change we settle on in the API changelog and `zulip.yaml`.

Reply:
Ahh, I see you have the docs in a later commit, but we still need to work out in "#api design" a compatibility plan to avoid breaking the mobile apps's markdown rendering.